app/views/articles/index.json.jbuilder

```diff
@@ -0,0 +1,13 @@
+docs = @presenter.documents.collect do |document|
+  link = document.eds_links.all.first # top priority one only
+  next if link.blank?
```

Comment:
Do we really want to not show a result if it doesn't return a link? (not sure if this is a real case or not).  Could this potentially have the affect of not returning the proper number of results (because some are omitted) or not returning EDS restricted results?

Reply:
thanks. i've added a test case for that and force pushed.